.gitignore

```diff
@@ -3,3 +3,4 @@
 composer.phar
 composer.lock
 .DS_Store
+.idea*
```

Comment:
Can you revert this change ? 

This change is related for a specific machine/os and not the project/package.

We encourage to use a global gitignore instread.